design/accepted/gc_plus/gc-plus-cloud.md

```diff
@@ -0,0 +1,52 @@
+# Managed Uncommitted Garbage Collection for lakeFS cloud - Execution Plan
+
+## Goals
+1. Run UGC for lakeFS cloud users
+2. Allow Cloud users to clean up storage space by deleting objects from the underlying storage for cost efficiency
+3. Support new and existing cloud users (both repository structures)
+
+## Non-Goals
+1. Remove lakeFS limitations during the UGC job run
+2. Improve UGC OSS support
+
+### User Story
+When creating a new cloud installation - on the GC initializations page,
+add UGC initialization (default run job=true for all repositories, default sweep = true).
+It can get the same rules as the GC.
```

Comment:
I don't think the committed GC retention rules are relevant for UGC

Reply:
right, I meant GC roles :)